### PR TITLE
DCS-898 added feedback survey banner

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

**Review by petergphillips — DISMISSED** [review]
LGTM

File: views/components/feedbackBanner/feedbackBanner.njk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="govuk-!-display-none-print">
+  <div class="govuk-width-container">
+    <div class="feedback-banner" data-qa="feedback-banner">
+      <a href="https://eu.surveymonkey.com/r/GYB8Y9Q?source={{ currentUrlPath }}" class="govuk-link govuk-link--inverse" target="_blank" rel="noopener noreferrer">
```
Comment (andrewrlee) [inline views/components/feedbackBanner/feedbackBanner.njk:4]: you need the host name in here - this is how the survey monkey questionnaire works out which service the feedback is for.

Worth having a look to see how this is done in PSH:
e.g: https://github.com/ministryofjustice/prisonstaffhub/blob/7c56f92fabfe712c4c599fb9e79eadd665b34deb/views/components/feedbackBanner/feedbackBanner.njk#L2

Comment (louisenorris) [reply views/components/feedbackBanner/feedbackBanner.njk]: Aha - thanks I missed this but all makes sense now, that bit had confused me a little.

File: views/components/feedbackBanner/feedbackBanner.njk [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="govuk-!-display-none-print">
+  <div class="govuk-width-container">
+    <div class="feedback-banner" data-qa="feedback-banner">
+      <a href="https://eu.surveymonkey.com/r/GYB8Y9Q?source={{ hostname + currentUrlPath }}" class="govuk-link govuk-link--inverse" target="_blank" rel="noopener noreferrer">
+        Give feedback on this service
+      </a>
+    </div>
+  </div>
+</div>
```
Comment (andrewrlee) [inline views/components/feedbackBanner/feedbackBanner.njk:9]: the little red thing  is saying you are missing a new line at t he end of the file

NBD but technically files without new lines at the end aren't valid files

bit annoying we don't have linting rules to check this

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**